types/jsonwebtoken/index.d.ts

```diff
@@ -12,6 +12,7 @@
 //                 Ivan Sieder <https://github.com/ivansieder>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 //                 Nandor Kraszlan <https://github.com/nandi95>
+//                 Jimmy Leung <https://github.com/hkleungai>
```

Comment:
Are you looking to become an owner/reviewer of this package? If you aren't, don't need to edit this block: https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
Issue addressed.